packages/flutter_tools/lib/src/commands/build_aar.dart

```diff
@@ -102,14 +104,17 @@ class BuildAarCommand extends BuildSubCommand {
       ? stringArg('build-number')
       : '1.0';
 
-    for (final String buildMode in const <String>['debug', 'profile', 'release']) {
+    const Map<String, BuildInfo> buildModes = {'debug': BuildInfo.debug, 'profile': BuildInfo.profile, 'release': BuildInfo.release};
```

Comment:
You're still using default values here, you'll need to explicitly construct a build info with all of the parameters set that you care about: https://github.com/flutter/flutter/blob/dd88b20435c1eb9b837bb4aa8b0a84aa0825530d/packages/flutter_tools/lib/src/build_info.dart#L14

Reply:
seems like that's what build_ios_framework is doing too. Does that one need refactoring too?

Reply:
🤷‍♂️